adcc/smoketests/test_smoke.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
```

Comment:
I would've just put this in the main folder as well or is there any reason I miss why this should be in a subfolder?